docs/source/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,11 @@
+{% extends "!autosummary/class.rst" %}
```

Comment:
I don't think this file should be commited? Also there's a bunch of svg files that probably don't need to go in (like the example tree sequence?)